firefox-ios/Client/Frontend/TrackingProtection/CertificatesCell.swift

```diff
@@ -0,0 +1,96 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import Common
+
+final class CertificatesCell: UITableViewCell, ReusableCell, ThemeApplicable {
+    var sectionLabel: UILabel = .build { label in
+        label.font = FXFontStyles.Bold.headline.scaledFont()
+        label.textAlignment = .right
+        label.numberOfLines = 0
+        label.lineBreakMode = .byWordWrapping
+        label.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
(See my other comment regarding the `.build` method). Same for line 84 in this file.